apps/fabric-website/src/pages/Styles/Colors/docs/web/ColorsImplementation.md

```diff
@@ -1,7 +1,7 @@
 ### Fluent UI React (JavaScript variables)
 
 ```jsx
-import { CommunicationColors } from '@uifabric/fluent-theme/lib/fluent/FluentColors';
+import { CommunicationColors } from '@fluentui/fluent-theme/lib/fluent/FluentColors';
```

Comment:
Probably don't change this yet--we should do an all-up pass for `@uifabric => @fluentui` renames **if/when** we decide to make that change, which I'm not sure if we have yet.